versioned/tiered/dynamodb/src/main/java/org/projectnessie/versioned/dynamodb/metrics/TracingExecutionInterceptor.java

```diff
@@ -76,11 +77,11 @@ public void afterMarshalling(final AfterMarshalling context,
     final Span span = executionAttributes.getAttribute(SPAN_ATTRIBUTE);
     final SdkHttpRequest httpRequest = context.httpRequest();
 
```

Comment:
nit: This class only exists while we are watiing for opentracing to update in Quarkus. Wonder if its worth maintaining cosmetic changes?

Reply:
ah, interesting. i've reverted the changes to this file.